rewrite-java/src/main/java/org/openrewrite/java/ChangeType.java

```diff
@@ -97,6 +105,16 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                     if (!Boolean.TRUE.equals(ignoreDefinition) && containsClassDefinition(cu, oldFullyQualifiedTypeName)) {
                         return SearchResult.found(cu);
                     }
+                    if (Boolean.TRUE.equals(visitStringLiterals)) {
+                        return Traits.literal().asVisitor(lit -> {
+                            stopAfterPreVisit();
+                            String string = lit.getString();
+                            if (string != null && string.contains(oldFullyQualifiedTypeName)) {
```

Comment:
Same here

Reply:
The `stopAfterPreVisit();` is only here and not in `ChangePackage`; any specific reason for that?

And same as above `reduce` can likely be used here as well.